core/src/main/kotlin/com/exactpro/th2/read/db/core/DataBaseService.kt

```diff
@@ -19,6 +19,8 @@ package com.exactpro.th2.read.db.core
 import kotlinx.coroutines.flow.Flow
 
 interface DataBaseService {
+    fun dataSourceHash(dataSourceId: DataSourceId): Int
+    fun queryHash(queryId: QueryId): Int
```

Comment:
This is not `DataBaseService` responsibility. I believe we should extract this into a separate interface and create a separate implementation for it